Cython/Compiler/Parsing.py

```diff
@@ -1539,9 +1545,23 @@ def p_expression_or_assignment(s):
                 rhs = p_testlist(s)
             return Nodes.InPlaceAssignmentNode(lhs.pos, operator=operator, lhs=lhs, rhs=rhs)
         expr = expr_list[0]
-        return Nodes.ExprStatNode(expr.pos, expr=expr)
+        if expr is None:
+            # i.e.: This should never happen unless we're in fault-tolerant mode.
+            # Change for an ellipsis so that other places expecting the expression
+            # work later on.
+            assert s.fault_tolerant
+            expr = ExprNodes.EllipsisNode(s.position())
```

Comment:
Since this is repeated (and might end up being repeated more often at some point), it seems worth a helper function.

BTW, what makes `expr` None here? I'm asking to find out if this is the right place to do the replacement, or if the functions that return the `None` here would be a better place.